doc/GenAIUsageGuidelines.md

```diff
@@ -0,0 +1,129 @@
+<!--
+Copyright IBM Corp. and others 2024
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution
+and is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following Secondary
+Licenses when the conditions for such availability set forth in the
+Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+version 2 with the GNU Classpath Exception [1] and GNU General Public
+License, version 2 with the OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] https://openjdk.org/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# Eclipse OMR Generative Artificial Intelligence Usage Guidelines
+
+The Eclipse OMR project acknowledges that some contributions may be authored in
+whole or in part from generative artificial intelligence (GenAI) coding
+assistants. Such contributions are permitted per the
+[Eclipse Foundation Generative AI Usage Guidelines](https://www.eclipse.org/projects/guidelines/genai/)
+which describes contributor and committer responsibilities, copyright and
+licensing considerations, and attribution requirements for GenAI assisted
+contributions.
+
+All contributors are encouraged to read and understand the advice and
+guidelines therein before submitting a contribution to Eclipse OMR even
+partially generated by an AI coding assistant.
+
+The Eclipse OMR project is subject to those policies and provides additional
+guidelines described below for contributors and committers governing
+GenAI-assisted contributions.
+
+## Attribution
+
+Contributions that include GenAI-assisted content must disclose that authorship
+in each file where GenAI content is present. Contributors must include a
+statement like the following just below the copyright header at the beginning
+of the file that acknowledges their AI assistant (assuming such a statement
+does not already exist).
+
+```
+// Some sections generated by <YOUR AI ASSISTANT>
+```
+
+In addition, individual commits that include GenAI authored content must
+attribute that at the end of the commit message (before any human authorship
+signatures).
+
+```
+Co-authored by <YOUR AI ASSISTANT>
+```
+
+## Tagging
+
+Some contributors’ employers may require their AI-generated code and
```

Comment:
Suggest using a plain ASCII apostrophe (unicode 0x27) instead of `’`.

Reply:
Fixed.